src/gui/editors/SongEditor.cpp

```diff
@@ -64,31 +62,21 @@ namespace lmms::gui
 
 namespace
 {
-constexpr int ZOOM_MIN = 15;
-constexpr int ZOOM_MAX = 1600;
-constexpr int ZOOM_INI = 100;
-}
 
-int scaleToLinearZoom(int val)
-{
-	// linear: val should be in min - max range.
-	float linVal = linearToLogScale(ZOOM_MIN, ZOOM_MAX, val);
-	return std::clamp(static_cast<int>(std::round(linVal)), ZOOM_MIN, ZOOM_MAX);
-}
+constexpr int MIN_PIXELS_PER_BAR = 2;
+constexpr int MAX_PIXELS_PER_BAR = 400;
+constexpr int ZOOM_STEPS = 200;
```

Comment:
`ZOOM_STEPS` is the number of possible steps in the slider. Due to the logarithmics this value doesn't mean anything special. It just affects the speed of zoom, since scrolling moves the slider by one step. So scrolling 200 steps goes from min to max zoom.